artistools/lightcurve.py

```diff
@@ -128,9 +128,16 @@ def get_magnitudes(modelpath):
         specdata = pd.read_csv(specfilename, delim_whitespace=True)
         timearray = specdata.columns.values[1:]
 
+    filters_dict = {}
+
+    filters_dict['bol'] = []
+    bolometric_magnitudes = bolometric_magnitude(modelpath, timearray)
+    for time, bol_magnitude in zip(timearray, bolometric_magnitudes):
+        if math.isfinite(bol_magnitude):
+            filters_dict['bol'].append((time, bol_magnitude))
```

Comment:
This can be made more compact (and faster, but that's probably irrelevant here) using a list comprehension.

```
filters_dict['bol'] = [
  (time, bol_magnitude)
  for time, bol_magnitude in bolometric_magnitude(modelpath, timearray)
  if math.isfinite(bol_magnitude)]
```